src/System.Windows.Forms/src/System/Windows/Forms/ListView.ListViewItemAccessibleObject.cs

```diff
@@ -0,0 +1,372 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Drawing;
+using System.Runtime.InteropServices;
+using Accessibility;
+using static Interop;
+
+namespace System.Windows.Forms
+{
+    public partial class ListView
+    {
+        /// <summary>
+        ///  ListBox control accessible object with UI Automation provider functionality.
+        ///  This inherits from the base ListBoxExAccessibleObject and ListBoxAccessibleObject
+        ///  to have all base functionality.
+        /// </summary>
+        internal class ListViewItemAccessibleObject : AccessibleObject
+        {
+            private readonly ListView _owningListView;
+            private readonly ListViewItem _owningItem;
+            private readonly ListViewGroup _owningGroup;
+            private bool DefaultGroupItem = false;
```

Comment:
Field names should start from `_` and have lower case first letter.
```suggestion
            private bool _defaultGroupItem = false;
```

Reply:
not relevant